proto-wine/ftnoir_protocol_wine_dialog.cpp

```diff
@@ -5,8 +5,10 @@
 #include <QDirIterator>
 #include <qdebug.h>
 #include <qdir.h>
+#include <qradiobutton.h>
```

Comment:
It's idiomatic in Qt to include `<QRadioButton>` even though your IDE suggests otherwise. The same issue is present in several other files.